tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -52,6 +52,7 @@ stages:
 
     variables:
       runCodesignValidationInjection: false
+      ComponentDetection.Timeout: 660
```

Comment:
Not sure if this needs to be specified in the top-level pipeline variables: 
https://github.com/microsoft/onnxruntime/blob/884acd4598a437521921dfdec596923afa3f4ed1/tools/ci_build/github/azure-pipelines/mac-react-native-ci-pipeline.yml#L48-L51